core/include/kernel/ldelf_syscalls.h

```diff
@@ -31,6 +31,11 @@ TEE_Result ldelf_syscall_map_bin(vaddr_t *va, size_t num_bytes,
 				 unsigned long flags);
 TEE_Result ldelf_syscall_copy_from_bin(void *dst, size_t offs, size_t num_bytes,
```

Comment:
We can remove `ldelf_syscall_copy_from_bin()` and its syscall entry if it's not used any longer. `ldelf` is always updated in sync with OP-TEE core so there's no need for a stable ABI.

Reply:
`ldelf_syscall_copy_from_bin()` is still used by `copy_remapped_to`

Reply:
OK